iBridgesCli.py

```diff
@@ -71,6 +71,9 @@ def __init__(self,                      # pylint: disable=too-many-arguments
         self.config_file = None
         self.download_finished = None
         self.upload_finished = None
+        self.irods_conn = None
+
+        default_irods_env = os.path.join(str(os.getenv('HOME')), '.irods', 'irods_environment.json')
```

Comment:
Not sure `os.getenv('HOME')` makes sense in a Windows environment. Maybe John's suggestion also takes care of that?